scripts/reindex.py

```diff
@@ -141,21 +145,30 @@ def conda_data_wrapper(args):
         print(f"{type(exc)}: {exc}", file=sys.stderr)
         active_pypi_versions = {}
 
+    # move the errors file to the top /public folder
+    desired_errors_path = PUBLIC / "errors.json"
+    current_error_path = Path(f"{PUBLIC}/manifest/errors.json")
+    if current_error_path.exists():
+        current_error_path.rename(desired_errors_path)
+    
     # load each manifest & build the indices (while verifying the manifest)
-    for mf_file in (PUBLIC / "manifest").glob("*.json"):
-        # move the errors file to top /public folder
-        if mf_file.name == "errors.json":
-            mf_file.rename(PUBLIC / "errors.json")
+    for normalized_name, info in active_pypi_versions.items():
+        name = info["name"]
+        pypi_versions = info["pypi_versions"]
+
+        mf_file = Path(f"{PUBLIC}/manifest/{normalized_name}.json")
+        if not mf_file.exists():
```

Comment:
```suggestion
        manifest_file = Path(f"{PUBLIC}/manifest/{normalized_name}.json")
        if not manifest_file.exists():
```

I would reduce the cognitive load and be explicit in the variable name.